regionprops tests isn't quite right

## Description
```
$ pytest skimage/measure/tests/test_regionprops.py::test_regionprops_table                               
Test session starts (platform: linux, Python 3.7.3, pytest 5.1.0, pytest-sugar 0.9.2)
rootdir: /home/mark/git/scikit-image/skimage/measure
plugins: forked-1.0.2, xdist-1.29.0, sugar-0.9.2
collecting ... 

―――――――――――――――――――――――――――――――――――――――― test_regionprops_table ―――――――――――――――――――――――――――――――――――――――――

    def test_regionprops_table():
        out = regionprops_table(SAMPLE)
        assert out == {'label': array([1]),
                       'bbox-0': array([0]), 'bbox-1': array([0]),
                       'bbox-2': array([10]), 'bbox-3': array([18])}
    
        out = regionprops_table(SAMPLE, properties=('label', 'area', 'bbox'),
                                separator='+')
>       assert out == {'label': array([1]), 'area': array([180]),
                       'bbox+0': array([0]), 'bbox+1': array([0]),
                       'bbox+2': array([10]), 'bbox+3': array([18])}
E       AssertionError: assert {'area': arra...ay([10]), ...} == {'area': arra...ay([10]), ...}
E         Omitting 5 identical items, use -vv to show
E         Differing items:
E         {'area': array([72])} != {'area': array([180])}
E         Use -v to get the full diff

skimage/measure/tests/test_regionprops.py:514: AssertionError

 tests/test_regionprops.py ⨯ 
```
fails

This seems to be because SAMPLE is modified???

Importing pdb at some point before things are executed shows that this is the case

```
collecting ... 
 tests/test_regionprops.py ✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓✓                 94% █████████▌
>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>> PDB set_trace (IO-capturing turned off) >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>
> /home/mark/git/scikit-image/skimage/measure/tests/test_regionprops.py(508)test_regionprops_table()
-> out = regionprops_table(SAMPLE)
(Pdb) SAMPLE
array([[1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
       [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
       [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
       [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
       [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
       [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
       [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
       [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
       [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1],
       [1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 1]])
```
## Way to reproduce
```python
# Place the full code we need to recreate your issue here
# upload all necessary images to github too!
```


## Version information

```python
Python 3.7.3 | packaged by conda-forge | (default, Jul  1 2019, 21:52:21) 
[GCC 7.3.0] :: Anaconda, Inc. on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> # Paste the output of the following python commands
... from __future__ import print_function
>>> import sys; print(sys.version)
3.7.3 | packaged by conda-forge | (default, Jul  1 2019, 21:52:21) 
[GCC 7.3.0]
>>> import platform; print(platform.platform())
Linux-5.0.0-23-generic-x86_64-with-debian-buster-sid
>>> import skimage; print("scikit-image version: {}".format(skimage.__version__))

scikit-image version: 0.16.dev0
>>> import numpy; print("numpy version: {}".format(numpy.__version__))
numpy version: 1.17.0

```

--- a/skimage/measure/tests/test_regionprops.py
+++ b/skimage/measure/tests/test_regionprops.py
@@ -451,7 +451,8 @@ def test_iterate_all_props():
 
 
 def test_cache():
-    region = regionprops(SAMPLE)[0]
+    SAMPLE_mod = SAMPLE.copy()
+    region = regionprops(SAMPLE_mod)[0]
     f0 = region.filled_image
     region._label_image[:10] = 1
     f1 = region.filled_image
@@ -498,7 +499,7 @@ def test_props_to_dict():
     regions = regionprops(SAMPLE)
     out = _props_to_dict(regions, properties=('label', 'area', 'bbox'),
                          separator='+')
-    assert out == {'label': array([1]), 'area': array([180]),
+    assert out == {'label': array([1]), 'area': array([72]),
                    'bbox+0': array([0]), 'bbox+1': array([0]),
                    'bbox+2': array([10]), 'bbox+3': array([18])}
 
@@ -511,7 +512,7 @@ def test_regionprops_table():
 
     out = regionprops_table(SAMPLE, properties=('label', 'area', 'bbox'),
                             separator='+')
-    assert out == {'label': array([1]), 'area': array([180]),
+    assert out == {'label': array([1]), 'area': array([72]),
                    'bbox+0': array([0]), 'bbox+1': array([0]),
                    'bbox+2': array([10]), 'bbox+3': array([18])}
 

